### DISCUSSION TITLE
Share to can append to existing issue

### DISCUSSION THREAD

**Discussion by suburbanbuddha** [community/community#43012] [Mobile]
Hi
I often use the **share to** feature on my phone to create GitHub issues

I wanted to ask if it might be in the works( since I didn't see it on the GitHub road map) to add this option when sharing to my GitHub mobile app - create new issue, add to existing issue

Simple example:

I'm learning about something and realize a feature related to it that I wanted to make so I do a shared to of the article as the start of my issue. As I do more research, I want to add more links so I would like to do a share to and select that issue again. Ideally, it would be great if the mobile app remembers the last few issues so at least I can pick from those before having to hunt for the issue with search.

Hope that makes sense

Thanks

**Comment by HenSquared** [comment]
Hey! Thanks for the feature request! These help us work to make GitHub Mobile better. 

To make sure I understand your request correctly, you'd like to be able to use share sheet to copy the URL of an article / webpage to a new issue? Do I have that right?

How would this populate on the issue? What would the title of the Issue look like? And the body? 

If we were to 'add to existing issue', what would be appended to the existing issue? Would we append the article URL?

As a workaround for now, you can copy text from the article or URL and manually paste into a new issue by long pressing GitHub app on your home screen from iOS.

**Reply by suburbanbuddha** [reply]
Hi @HenSquared,

Not exactly; the "Share to" menu on mobile devices already allows the creation of a GitHub issue when browsing - i.e.: I own a Google Pixel 6 with GitHub Mobile installed, open Google Chrome, navigate to a webpage that explains how to set up continuous delivery, click "Share", pick "Create Issue", which opens GitHub Mobile and prompts me to select the repository that the new issue will be created in.

What I'd like added is: 
1. I own a Google Pixel 6 with GitHub Mobile installed
2. Open Google Chrome
3. Navigate to a webpage that explains how to set up continuous delivery
4. Click "Share" 
5. I have the option to pick "Create Issue" or "Append to Issue"
6. I choose "Append to Issue", which opens GitHub Mobile, prompts me to select the repository, and finally the issue by title, which I start typing, filtering the list of issues down until the one I want to append to appears, so I can select it.

I hope this clarifies my feature request sufficiently. 

I'm already using a manual work around - i.e.: I copy the URL and then open GitHub Mobile manually to search for the existing issues, and paste to its description, or add it as a comment.

**Reply by suburbanbuddha** [reply]
PS: Apologies for delay following up, and thanks for your time!
